applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -203,6 +204,14 @@ config STREAM_BIDIRECTIONAL
 	  Bidirectional stream enables encoder and decoder on both sides,
 	  and one device can both send and receive audio.
 
+config WALKIE_TALKIE_DEMO
+	select STREAM_BIDIRECTIONAL
```

Comment:
Why is "select" used for STREAM_BIDIRECTIONAL in stead of "default y if WALKIE_TALKIE_DEMO"?

Reply:
If there are no specific reasons I suggest changing it to "default y if WALKIE_TALKIE_DEMO" to keep consistency

Reply:
It's a matter of what settings knows about what. The walkie-talkie demo should know there is  a bidirectional setting and needs it, but bidirectional doesn't needs to know there is a walkie-talkie demo. Also, this will catch any changes to stream_bidirectional (if it is removed), but the other way around has no such guard.

Reply:
Also, I couldn't use the same way to select AUDIO_SOURCE_I2S and TRANSPORT_CIS because they are choices. Maybe @tejlmand knows how this should be done?